app/views/Project/Tagging/Sources/SourcesFilter/EntryFilter/index.tsx

```diff
@@ -0,0 +1,202 @@
+import React, { useMemo, useCallback } from 'react';
+import {
+    _cs,
+    listToMap,
+} from '@togglecorp/fujs';
+import {
+    DateRangeInput,
+    MultiSelectInput,
+    SelectInput,
+    List,
+} from '@the-deep/deep-ui';
+import {
+    useFormArray,
+    useFormObject,
+    SetValueArg,
+} from '@togglecorp/toggle-form';
+import {
+    SourceFilterOptionsQuery,
+    AnalysisFrameworkFilterType,
+} from '#generated/types';
+import {
+    hasNoData,
+    enumKeySelector,
+    enumLabelSelector,
+} from '#utils/common';
+import ProjectMemberMultiSelectInput from '#components/selections/ProjectMemberMultiSelectInput';
+import BooleanInput, { Option } from '#components/selections/BooleanInput';
+
+import FrameworkFilterItem from './FrameworkFilterItem';
+import { PartialEntriesFilterDataType } from '..';
+import styles from './styles.css';
+
+const filterKeySelector = (d: AnalysisFrameworkFilterType) => d.key;
+
+const controlledStatusOptions: Option[] = [
+    {
+        key: 'true',
+        value: 'Controlled',
+    },
+    {
+        key: 'false',
+        value: 'Not controlled',
+    },
+];
```

Comment:
Shouldn't we use BooleanInput

Reply:
We are using BooleanInput. The options are sent from the parent component